jquery-src/jquery.calendars.chinese.js

```diff
@@ -273,7 +273,7 @@
             var monthDaysTable = LUNAR_MONTH_DAYS[year - LUNAR_MONTH_DAYS[0]];
             var intercalaryMonth = monthDaysTable >> 13;
 
-            return (intercalaryMonth && intercalaryMonth === monthIndex);
+            return (intercalaryMonth !== 0 && intercalaryMonth === monthIndex);
```

Comment:
"mostly"? If 0 is not a possible intercalary month (which looks to me like a true statement), then returning something falsy (might as well be zero) or the month if there is one seems more useful, yes.